app/models/post.rb

```diff
@@ -18,4 +18,13 @@ def clickbait?
     return unless CLICKBAIT_PATTERNS.none? { |pat| pat.match title }
     errors.add(:title, "must be clickbait")
   end
+
+  FactoryBot.define do
```

Comment:
In the future, put factories in `spec/factories`.  Test logic is never the concern of the application (inside `app/`).